src/scales/scale.logarithmic.js

```diff
@@ -64,13 +64,14 @@ var defaultConfig = {
 	}
 };
 
-// TODO(v3): change this to positiveOrDefault
-function nonNegativeOrDefault(value, defaultValue) {
+function positiveOrDefault(value, defaultValue) {
```

Comment:
`nonNegative` is a more accurate description than `positive`. I think I might only change the wording `positive` if we also change `>=` to `>` (which maybe would be a better check since `log 0` is `-Infinity` - though I didn't check all the places it was being called to see if that's a valid thing to do)

Reply:
How about getting rid of the whole shebang? 💥